packages/table/src/common/utils.ts

```diff
@@ -422,94 +337,50 @@ export const Utils = {
         return event.button === 0;
     },
 
+    // these functions used to live here but now live in core. since these Utils
+    // are in the public API, we provide facades here - complete with function
+    // descriptions - so as to make the refactor invisible externally.
+
     /**
-     * Returns true if the arrays are equal. Elements will be shallowly compared by default, or they
-     * will be compared using the custom `compare` function if one is provided.
+     * Returns true if the arrays are equal. Elements will be shallowly compared
+     * by default, or they will be compared using the custom `compare` function
+     * if one is provided.
+     * @deprecated since 1.26.0; import this function from core Utils instead.
```

Comment:
@llorca if this PR makes it into this week's release, I believe this table version would be correct, right?

Reply:
Correct! Can we explicitly say `@deprecated since **@blueprintjs/table 1.26.0**` (not sure if we have a pattern for that) to avoid confusion with the generic/core version?